model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,55 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - ref: os.name
+        requirement_level: "required"
+        brief: >
+            This attribute indicates the operating system name of the device on which the crash event occurred.
+        examples: ['iOS', 'Android']
+      - ref: ios.state
+        requirement_level: recommended
+        brief: >
+          This attribute represents the state the application has transitioned into at the occurrence of the event
+          For ios crashes
+        examples: ['active', 'inactive', 'background', 'foreground', 'terminate']
+      - ref: android.state
+        requirement_level: recommended
+        brief: >
+          This attribute represents the state the application has transitioned into at the occurrence of the event
+          For android crashes
+        examples: ['created', 'foreground', 'background']
+      - ref: exception.stacktrace
+        requirement_level: recommended
+        brief: >
+          The contents of printStackTrace() being called on the throwable
+          For android crashes
+      - ref: thread.id
+        requirement_level: recommended
+        brief: >
+          The id of the thread as output by Thread.currentThread().getId()
+          For android crashes
+      - ref: thread.name
+        requirement_level: recommended
+        brief: >
+          The name of the thread as output by Thread.currentThread().getName()
+          For android crashes
+      - id: exception.message
+        requirement_level: recommended
+        brief: >
+          The message within the exception object
+          For android crashes
+        type: string
+        examples: ['java.lang.ArithmeticException: / by zero']
+      - id: ios.crash_report
+        requirement_level: recommended
+        brief: >
+          The contents from PLCrashReporter
+          Contains three primary elements. "images", "threads" and "signal". These could be broken out as individual elements
```

Comment:
What is meant here by "These could be broken out as individual elements"?

Do you mean put them into separate attributes?

Reply:
Images and Threads are arrays containing details of objects linked and stack traces respectively.
The signal elements are broken out separately.   I will add a sample to the previous comment